tests/test.zig

```diff
@@ -685,6 +686,46 @@ test "parse - int" {
     });
 }
 
+test "parse - int (custom type)" {
+    const Foo = struct {
+        int: i32,
+
+        pub const @"getty.db" = struct {
+            pub fn deserialize(
+                ally: std.mem.Allocator,
+                comptime _: type,
+                deserializer: anytype,
+                visitor: anytype,
+            ) @TypeOf(deserializer).Err!@TypeOf(visitor).Value {
+                return try deserializer.deserializeInt(ally, visitor);
+            }
+
+            pub fn Visitor(comptime Value: type) type {
+                return struct {
+                    pub usingnamespace getty.de.Visitor(@This(), Value, .{ .visitInt = visitInt });
```

Comment:
nit: Could you make the arguments here go on their own lines? Just to be consistent with the rest of the codebase.